### ISSUE TITLE
[Techstack-beta] Make prompts unselectable

### ISSUE THREAD

**Issue by jtojnar** [NixOS/nixos-homepage#1380]
**Describe the glitch**

The prompts on the [download page](https://nixos.org/download/) regressed yet again. They can be selected, which prevents easy copying. Previously:

- https://github.com/NixOS/nixos-homepage/issues/1094
- https://github.com/NixOS/nixos-homepage/pull/492

Also in Nixpkgs:

- https://github.com/NixOS/nixpkgs/pull/301439
- https://github.com/NixOS/nixpkgs/pull/77683

**Additional context**

It needs [`user-select: none`](https://developer.mozilla.org/en-US/docs/Web/CSS/user-select) CSS property.

**Comment by thilobillerbeck** [comment]
Good point, just anyone grabs this, since we process most of the prompts from markdown, we may can introduce a e.g. remark plugin which processes bash code snippets in such a way, that it has the structure to readily apply the non selectable properties.

**Comment by thilobillerbeck** [comment]
Good point, just anyone grabs this, since we process most of the prompts from markdown, we may can introduce a e.g. remark plugin which processes bash code snippets in such a way, that it has the structure to readily apply the non selectable properties.

Maybe the highlighter also added some consistent naming to the prompt, although we should check upfront how much this is prone to break in the future, since it is clearly not meant to be hooked into

**Comment by thilobillerbeck** [comment]
fixed by 86c0036534dcca30b93f7410f5ec2803f029d4e8 and aadeecd400fc4d9e722e8c9b6435823e661025d9
